__tests__/src/html.js

```diff
@@ -9,4 +9,12 @@ describe('html', () => {
     expect(str.indexOf(title) !== -1).toBeTruthy()
     expect(str).toMatchSnapshot()
   })
+
+  it('should return an HTML string with a given staticHost', () => {
+    const staticHost = 'https://fakeHost.com/'
```

Comment:
@evansiroky domain names are not case sensitive but tests will fail in `nock` and other comparisons if we use mixed-cases so I would recommend keeping all domain names lowercase in tests.